### PR TITLE
[JENKINS-60866] Move some inline JS out into separate files

### PR DESCRIPTION
**Pull request by daniel-beck**
See [JENKINS-60866](https://issues.jenkins-ci.org/browse/JENKINS-60866).

### Proposed changelog entries

(too minor? Or due to regression risk "Internal JavaScript resource cleanup"?)

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/lib/hudson/newFromList/form.jelly
Comment (Wadeck) [inline core/src/main/resources/lib/hudson/newFromList/form.jelly:97]: **(not related to this PR)**

`style="margin-left:5em"`

I was not able to understand why this is done... I imagine it's a forgotten debug utility used during development. Once the makeButton is executed, the Ok button is moved by 5em to the left due to the removal of the margin-left. If you put a breakpoint before the `makeButton(...)` you will see the Ok with a space.
Does not really impact the end user, but it's fairly confusing for not apparent reasons. It was there since the beginning, in 2009 🕵️ Perhaps at the age of IE (50+% of market share), the display:none of the radio was problematic 🤷‍♂️

**Review by Wadeck — CHANGES_REQUESTED** [review]
Manually tested:
- ✔️ Install tab in the PluginManager, checking the box makes the notification message appears
- ✔️ View looking at the screenResolution cookie
- ✔️ AdminMonitor footer2header script, checked the order is the same as before (first the anonymous function then the `DOMContentLoaded`)
- ✔️ Running a build, looking at the console, the icon is changed when the build is done
- ✔️ Widgets executors and buildQueue are refreshed as before (no impact on buildHistory from job pages)
- ✔️ Creating a view using the newFromList tag
- ✔️ Search bar still working with autocompletion
- ❌ Stop button did not work everytime. I got some times the button clicks being not caught, leading to a 405 page. Not able to reproduce in a deterministic way (only spent 20min), it's a matter of finding where to put breakpoints I imagine.

_(at the same time, as I cannot reproduce the case easily, I have some doubts that it's just my local setup :D)_

**Review by Wadeck — APPROVED** [review]
(potentially) Problematic code was removed. The rest was tested, so it's fine to move forward :)

(sorry for the delay)

**Comment by daniel-beck** [comment]
> The rest was tested

FTR https://github.com/jenkinsci/jenkins/pull/5511/commits/7d687159d041507749586cc4dbfdb4396c0d9243 was a nontrivial merge, and I only tested with `secure == false`.

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by jtnord — APPROVED** [review]
lgtm

**Comment by timja** [comment]
conflicted

**Comment by daniel-beck** [comment]
Removed the conflicted parts.

**Comment by timja** [comment]
More conflicts 😢

**Comment by daniel-beck** [comment]
I give up.

**Pull request closed without merging**
